cmd/prometheus/main.go

```diff
@@ -174,11 +183,13 @@ type Reloadable interface {
 
 func reloadConfig(filename string, rls ...Reloadable) bool {
 	log.Infof("Loading configuration file %s", filename)
+	configAttempts.WithLabelValues("attempted").Add(1)
```

Comment:
Yes, the common approach to those tried/failed/succeeded metrics is this:
https://github.com/prometheus/prometheus/blob/master/retrieval/discovery/dns.go#L42-L53